buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -56,7 +56,7 @@ object Versions {
 
     const val constraintlayout: String = "2.0.4"
 
-    const val fragment_ktx: String = "1.2.5"
+    const val fragment_ktx: String = "1.3.0-beta02"
```

Comment:
I was going to say to mention why this has changed, but maybe not necessary 🤷

Reply:
Well, just in case:

The functionality used in the PermissionManager for registering the fragment to the permission result is only in the beta version of the library